versioned/storage/bigtable/src/main/java/org/projectnessie/versioned/storage/bigtable/BigTableBackend.java

```diff
@@ -166,8 +168,16 @@ private boolean eraseRepositoriesAdminClient(Set<String> repositoryIds) {
 
     for (String repoId : repositoryIds) {
       ByteString prefix = copyFromUtf8(repoId + ':');
-      tableAdminClient.dropRowRange(tableRefs, prefix);
-      tableAdminClient.dropRowRange(tableObjs, prefix);
+      try {
+        tableAdminClient.dropRowRange(tableRefs, prefix);
+        tableAdminClient.dropRowRange(tableObjs, prefix);
+      } catch (ResourceExhaustedException e) {
+        LOGGER.warn("DropRowRange quota exceeded, trying the non-admin path", e);
+        return false;
```

Comment:
I think we don't want to re-delete the repo-Ids that have already been processed here. WDYT?